data/hai/unfettered 2 war preparations.txt

```diff
@@ -0,0 +1,2171 @@
+
+
+
+# 18 is optional, in side missions because you can learn Hai through the Hai diplomats
+# idea: add a duel against the Herald where they have a Water Bug wearing an Octopulse
+mission "<uhai> 19: Scout Wanderer Systems"
+	name `Know your enemy`
+	description `Go to <waypoints> and scout the fleet presence of all Wanderer ships. Then, come back to <destination> for your reward.`
+	source
+		attributes "unfettered"
+	destination "Warfeed"
+	to offer
+		has "language: Hai"
+		has "<uhai> 12: Outfitter Upgrade: done"
+		not "<uhai> 18: Hai Language: active"
+	on offer
+		log "Minor People" "Mysterious Herald" "A Herald of He-Yi that does not want to reveal their true identity. They seem to have a lot of culture, and stay calm in most situations."
+		conversation
+			`You exit your ship and head to the spaceport as usual, but you cannot shake the sensation that you are being followed. Looking around you see a curious figure with a bird mask: the Herald. They head your way, aware of the fact they have been spotted.`
+			`	As they get closer you feel like the color of the mask has slightly changed, but their voice seems the same "It did not take you long to spot me, did it? You have bested me<, unfettered title> <last>!" They say, bowing their head so little you barely notice it. "Rest assured, I am not just here to watch over you; my master has yet another task to ask of you before the war starts."`
+			choice
+				`	"I am listening."`
+					goto task
+				`	"Why were you watching me?"`
+				`	"First tell me, why have you not introduced yourself yet?"`
+					goto masked
+				`	"Later."`
+					defer
+			label watching
+			action
+				set "uhai choice: watching"
+			`	They look around discreetly - more so than earlier - before speaking in a lower tone, "Come, let us have a walk." You follow them, and they continue talking on the way, "Why does the To'or fight to the death? Is it pride? Is it protection of their loved ones, or a matter character? There are often multiple reasons for why things happen the way they do, and we do not always know them all."`
+			`	They then look at you directly, "You should know that there are some among us that do not appreciate the status you have been given, or what happened with the Alphas."`
+			choice
+				`	"What was the task again?"`
+					goto task
+				`	"Can you tell me who I should be afraid of?"`
+				`	"What is a To'or?"`
+					to display
+						not "unfettered: to'or knowledge"
+					goto toor
+			action
+				set "unfettered: warned about scarred"
+			`	"You should fear the Scarred. I will say no more, for information has a price."`
+				goto task
+			label toor
+			action
+				set "unfettered: to'or knowledge"
+			`	"It is the most mighty creature we know of. They live on Hai-home, in small but resilient communities, where every member has earned their place, and where nobody is left behind. They never run away from a fight, and rightly so, since they are the most impressive creatures on the planet."`
```

Comment:
Interesting to note that the Hai are less impressive.

Reply:
I imagine these as carapaced monsters that are strong so yeah def more impressive